src/widget/base-widget.js

```diff
@@ -9,7 +9,7 @@ import { openDialog } from './dialog'
 
 class BaseWidget {
   constructor(element, settings) {
-    this.element = element
+    this.element = $(element)
```

Comment:
why are you return jQuery here?

Reply:
because below `this.element` used as jquery element, but now it native dom element